pkg/apis/cyclone/v1alpha1/workflow_trigger.go

```diff
@@ -30,21 +30,21 @@ const (
 
 // WorkflowTriggerSpec defines workflow trigger definition.
 type WorkflowTriggerSpec struct {
-	// Reference of the Workflow to trigger
-	WorkflowRef *corev1.ObjectReference `json:"workflowRef"`
 	// Type of this trigger, Schedule or Webhook
 	Type TriggerType `json:"triggerType"`
-	// Whether this trigger is enabled, if set to true, no workflow will be triggered
-	Enabled bool `json:"enabled"`
```

Comment:
Could you please explain why remove this `Enabled` ?

Reply:
Put to status section

Reply:
I think `Enabled` is a *config* of the trigger, not a *status*. So maybe put it in `spec` is better. WDYT?